docs/content/en/docs/topics/restore-a-backup/_index.md

```diff
@@ -32,7 +33,20 @@ You will need storage for the backups. This topic shows the use of AWS S3 bucket
 
 * If you'll use AWS S3, before proceeding with the configuration described below, verify that you know the `aws_access_key_id` and `aws_secret_access_key` values. Or  contact your IT team if they manage those assets. You'll provide those details in an edited version of the [medusa-bucket-key.yaml](./medusa-bucket-key.yaml) file. For information about the S3 setup steps, see this helpful [readme](https://github.com/thelastpickle/cassandra-medusa/blob/master/docs/aws_s3_setup.md).  
 
-* If you haven’t already, install the k8ssandra chart.
+* If you haven't already, add and update the following repo, which has in one chart all the settings for K8ssandra plus the backup/restore settings:
+
+```
+helm repo add k8ssandra https://helm.k8ssandra.io/
+helm repo update
+```
+
+```
+Hang tight while we grab the latest from your chart repositories...
+...Successfully got an update from the "k8ssandra" chart repository
+Update Complete. ⎈Happy Helming!⎈
+```
+
+* If you haven’t already, install the k8ssandra chart in any namespace; this command will use the `default` namespace.
```

Comment:
`If you haven’t already,`

Could we rephrase this as it is the same as the previous item? I'm on the fence regarding this change request so feel free to push back.

Reply:
Done, thanks